tests/app/core/models/session-model.spec.ts

```diff
@@ -0,0 +1,59 @@
+import { assertSessionDataIntegrity } from '../../../../src/app/core/models/session.model';
+import { of } from 'rxjs';
+
+const validSessionData = {
+  openPhases: [
+    'phaseBugReporting',
+    'phaseTeamResponse',
+    'phaseTesterResponse',
+    'phaseModeration',
+  ],
+  phaseBugReporting: 'bugreporting',
+  phaseTeamResponse: 'pe-results',
+  phaseTesterResponse: 'testerresponse',
+  phaseModeration: 'pe-evaluation',
+};
```

Comment:
I think as mentioned in https://github.com/CATcher-org/CATcher/pull/560#discussion_r566606895,
maybe we can use `Phase.phaseBugReporting` and the others as the keys for the `validSessionData` as well as the values in the `openPhases` array.